tests/unit/common.py

```diff
@@ -0,0 +1,17 @@
+"""
+Common functions for tests
+"""
+
+
+def get_payload(post_message):
```

Comment:
I would say this belongs in a fixture. It can be added to conftest.py to allow sharing.
https://docs.pytest.org/en/latest/fixture.html#scope-sharing-a-fixture-instance-across-tests-in-a-class-module-or-session

Reply:
ok